.remarkignore

```diff
@@ -0,0 +1 @@
+site/_includes/partials/devtools/*/whats-new.md
```

Comment:
@matthiasrohmer I took your advise to use 11ty/nunjucks syntax. However, I got a warning from the `lint:md` npm task. 11ty syntax is not allowed (see `no-unescaped-template-tags.js`).  

I checked the `remark-cli` docs, I couldn't find a way to disable the warning for just a line (e.g. `eslint-disable-next-line`). The documentation suggests me to ignore the files in `.remarkignore`. 

At this point, I can either fallback to the previous regex version or ignore this file. Your call.

Reply:
Is this still valid? Now all templates seem to live in tools/devtools/templates. Or is remark complaining after generation?